apps/site/src/app/(home)/page.tsx

```diff
@@ -1,7 +1,18 @@
 import { Landing } from "./sections";
+import FAQ from "./sections/FAQ";
+
+export const revalidate = 60;
 
 export default function Home() {
 	// Show landing section only if still in maintenance,
 	// otherwise show the rest of the sections
-	return process.env.MAINTENANCE_MODE_HOME ? <Landing /> : <Landing />;
+	return process.env.MAINTENANCE_MODE_HOME ? (
+		<Landing />
+	) : (
+		<>
+			<Landing />
+			{/* @ts-expect-error Async Server Component */}
```

Comment:
Question: Is this `@ts-expect-error` line necessary?

Reply:
nope after removing the type prop from FAQ